### PR TITLE
test new workflow cron

### PR DESCRIPTION
**Pull request by Aricg**
Signed-off-by: Aric Gardner <agardner@linuxfoundation.org>

### CODE REVIEW

**Comment by Aricg** [comment]
test

**Comment by Aricg** [comment]
# Test Message
You must ask the current committer listed in the INFO file
to approve this change, once enough approvals are submitted
the pull request will automatically submit..

Note that there are no secrets exposed in this test...

<details>
<summary>PR to merge master 7a3f0c425df83c4e62716c4d5f750e22fef274ce -> master</summary>

```
AGENT_TOOLSDIRECTORY=/opt/hostedtoolcache
ANDROID_HOME=/usr/local/lib/android/sdk
ANDROID_SDK_ROOT=/usr/local/lib/android/sdk
ANT_HOME=/usr/share/ant
AZURE_EXTENSION_DIR=/opt/az/azcliextensions
BOOST_ROOT_1_69_0=/usr/local/share/boost/1.69.0
BOOST_ROOT_1_72_0=/usr/local/share/boost/1.72.0
CHROMEWEBDRIVER=/usr/local/share/chrome_driver
CHROME_BIN=/usr/bin/google-chrome
CI=true
CONDA=/usr/share/miniconda
DEBIAN_FRONTEND=noninteractive
DEPLOYMENT_BASEPATH=/opt/runner
DOTNET_SKIP_FIRST_TIME_EXPERIENCE="1"
GECKOWEBDRIVER=/usr/local/share/gecko_driver
GITHUB_ACTION=run3
GITHUB_ACTIONS=true
GITHUB_ACTOR=Aricg
GITHUB_BASE_REF=master
GITHUB_EVENT_NAME=pull_request
GITHUB_EVENT_PATH=/home/runner/work/_temp/_github_workflow/event.json
GITHUB_HEAD_REF=master
GITHUB_JOB=test
GITHUB_REF=refs/pull/9/merge
GITHUB_REPOSITORY=lfit-sandbox/self-service-committer-management
GITHUB_REPOSITORY_OWNER=lfit-sandbox
GITHUB_RUN_ID=116103092
GITHUB_RUN_NUMBER=13
GITHUB_SHA=7a3f0c425df83c4e62716c4d5f750e22fef274ce
GITHUB_WORKFLOW=Create PR message
GITHUB_WORKSPACE=/home/runner/work/self-service-committer-management/self-service-committer-management
GOROOT=/opt/hostedtoolcache/go/1.14.3/x64
GOROOT_1_11_X64=/opt/hostedtoolcache/go/1.11.13/x64
GOROOT_1_12_X64=/opt/hostedtoolcache/go/1.12.17/x64
GOROOT_1_13_X64=/opt/hostedtoolcache/go/1.13.11/x64
GOROOT_1_14_X64=/opt/hostedtoolcache/go/1.14.3/x64
GRADLE_HOME=/usr/share/gradle
HOME=/home/runner
HOMEBREW_CELLAR="/home/linuxbrew/.linuxbrew/Cellar"
HOMEBREW_PREFIX="/home/linuxbrew/.linuxbrew"
HOMEBREW_REPOSITORY="/home/linuxbrew/.linuxbrew/Homebrew"
INVOCATION_ID=a581af1f35a9499a8d2f1b41cf582802
ImageOS=ubuntu18
ImageVersion=20200518.1
JAVA_HOME=/usr/lib/jvm/zulu-8-azure-amd64
JAVA_HOME_11_X64=/usr/lib/jvm/zulu-11-azure-amd64
JAVA_HOME_12_X64=/usr/lib/jvm/zulu-12-azure-amd64
JAVA_HOME_7_X64=/usr/lib/jvm/zulu-7-azure-amd64
JAVA_HOME_8_X64=/usr/lib/jvm/zulu-8-azure-amd64
JOURNAL_STREAM=9:31515
LANG=C.UTF-8
LEIN_HOME=/usr/local/lib/lein
LEIN_JAR=/usr/local/lib/lein/self-installs/leiningen-2.9.3-standalone.jar
M2_HOME=/usr/share/apache-maven-3.6.3
PATH=/opt/hostedtoolcache/Python/3.6.10/x64/bin:/opt/hostedtoolcache/Python/3.6.10/x64:/usr/share/rust/.cargo/bin:/home/runner/.config/composer/vendor/bin:/home/runner/.dotnet/tools:/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin
PERFLOG_LOCATION_SETTING=RUNNER_PERFLOG
POWERSHELL_DISTRIBUTION_CHANNEL=GitHub-Actions-ubuntu18
PWD=/home/runner/work/self-service-committer-management/self-service-committer-management
RUNNER_OS=Linux
RUNNER_PERFLOG=/home/runner/perflog
RUNNER_TEMP=/home/runner/work/_temp
RUNNER_TOOL_CACHE=/opt/hostedtoolcache
RUNNER_TRACKING_ID=github_4dabad03-8cec-4f65-a22b-356ab842b28b
RUNNER_USER=runner
RUNNER_WORKSPACE=/home/runner/work/self-service-committer-management
SELENIUM_JAR_PATH=/usr/share/java/selenium-server-standalone.jar
SHLVL=1
SWIFT_PATH=/usr/share/swift/usr/bin
USER=runner
VCPKG_INSTALLATION_ROOT=/usr/local/share/vcpkg
_=/usr/bin/env
pythonLocation=/opt/hostedtoolcache/Python/3.6.10/x64
```

</details>
MERGEABLE: True Approvals: ['Aricg', 'bramwelt', 'vanessavalderrama']

**Review by bramwelt — APPROVED** [review]

**Review by vvalderrv — APPROVED** [review]

**Pull request merged**
